src/RpcServer/FixedDictionary.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+
+namespace Neo.Plugins
+{
+    public class FixedDictionary<TKey, TValue>
```

Comment:
Is this the `FIFOCache`?

https://github.com/neo-project/neo/blob/master/src/neo/IO/Caching/FIFOCache.cs

Reply:
FIFOCache is defined as internal,I'm afraid we cannot directly use it in rpcserver.

Reply:
Please consider using KeyedCollection.

https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.keyedcollection-2

Reply:
Already done, please review it again.